app/events/adjustment_event.rb

```diff
@@ -4,7 +4,7 @@ def self.publish(adjustment)
     create(
       eventable: adjustment,
       organization_id: adjustment.organization_id,
-      event_time: Time.zone.now,
+      event_time: adjustment.created_at,
```

Comment:
The passed in adjustment from AdjustmentCreateService, weirdly, could be an exisiting Adjustment rather than a freshly created one. Maybe so this gets a bit weird with that case; I'm not sure if `adjustment.updated_at` would be better or not.

Reply:
Adjustments are never updated, just created.

Reply:
OK for this one, I guess `AdjustmentCreateService` initialize we should sometime take out the first bit where it can be handed an Adjustment -- I verified that nothing does call it with one.